sonar-project.properties

```diff
@@ -6,7 +6,7 @@ sonar.projectName=SpaceToStudy-Client
 # Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
 
 sonar.sources=.
-sonar.exclusions=**/tests/**/*.js, **/src/containers/student-home-page/popular-categories/*.js, **/src/stories/*
+sonar.exclusions=**/tests/**/*.js, **/src/containers/student-home-page/popular-categories/*.js, **/src/stories/*, **/src/constants/**/*.js'
```

Comment:
If constants have any logic. It should be tested.

Reply:
ok